Passport/Passport.cpp

```diff
@@ -0,0 +1,117 @@
+#include <iostream>
+#include <iomanip>
+#include "Passport.h"
+
+#define SIZE 3
+#define MAX_SERIAL 999999
+
+using namespace std;
+
+char Passport::newSeries[SIZE] = {'A', 'A', 0};
```

Comment:
If you use initialisation like `array = { ... }` you don't have specify array's size explicily